src/lib/utils/urlUtils.ts

```diff
@@ -5,3 +5,5 @@ const WHITELISTED_URLS: string[] = [
 ];
 
 export const isWhitelistedUrl = (url: string): boolean => WHITELISTED_URLS.includes(url);
+
+export const getRemoveMemberCheckDetailsFullUrl = (id: string): string => `${constants.LANDING_URL}/${constants.REMOVE_MEMBER_PAGE}/${id}`;
```

Comment:
Please add unit tests in an appropriate place to test this function.

Reply:
I have added unit test for this function. Please re-review.